asketch2sketch.sketchplugin/Contents/Sketch/asketch2sketch.js

```diff
@@ -130,27 +130,27 @@ module.exports = console
 /* 1 */
 /***/ (function(module, exports) {
 
-var g;
```

Comment:
Is there any reason to update this file? I don't think there were any changes to the plugin.

Also, it looks like your version is in ES6+ while the previous one was ES5 - not sure if we should care about that? AFAIK Sketch plugins run inside of Safari's jscore which nowadays does support modern JS, but this really depends on the version of Safari user has. Not sure if users using older versions of macOS keep getting Safari updates 🤔 WDYT?